packages/console/src/components/PermissionsTable/index.tsx

```diff
@@ -236,12 +236,21 @@ function PermissionsTable({
         onRetry={retryHandler}
       />
       {editingScope && (
-        <EditPermissionModal
+        <EditScopeModal
+          scopeName={editingScope.name}
           data={editingScope}
+          text={{
+            title: 'permissions.edit_title',
+            nameField: 'api_resource_details.permission.name',
+            descriptionField: 'api_resource_details.permission.description',
+            descriptionPlaceholder: 'api_resource_details.permission.description_placeholder',
+          }}
+          onSubmit={async (editedData) => {
+            await handleEdit(editingScope, editedData);
```

Comment:
await seems unnecessary

Reply:
We need to wait the submit being handled before we close the modal.
<img width="521" alt="image" src="https://github.com/logto-io/logto/assets/10806653/8ee33290-9ea0-46ff-99f6-e56eb50a4a5c">